bedrock/legal_docs/views.py

```diff
@@ -25,10 +26,16 @@ def load_legal_doc(doc_name, locale):
 
     :param doc_name: name of the legal doc folder
     :param locale: preferred language version of the doc
-    :return: string content of the file or None
+    :return: dict containing string content of the file (or None), a boolean
+             value indicating whether the file is localized into the specified
+             locale, and a dict of all available locales for that document
     """
-    source = path.join(LEGAL_DOCS_PATH, doc_name, locale + '.md')
+    dir = path.join(LEGAL_DOCS_PATH, doc_name)
```

Comment:
This is shaddowing the builtin function `dir`. Make it a more explicit name to avoid confusion.